README.rst

```diff
@@ -64,14 +62,13 @@ Within your python file or notebook:
 - either for single regions
 
 .. code-block:: python
-   :linenos:
 
     query = Query.region('01')
 
 - or for all subregions a region (e.g. all Kommunen in a Bundesland)
+(to be implemented in Version 0.1.2)
```

Comment:
Not sure whether this is a bug or a feature. (Our first bug feature discussion XD)
Feature would mean its added in 0.2.0

Reply:
i would say bug, because its already in the docu of the Query